### DISCUSSION TITLE
lora of sd3

### DISCUSSION THREAD

**Discussion by jiahui-w** [huggingface/diffusers#9632] [Q&A]
How do StableDiffusion3Pipeline load lora models？

**Comment by a-r-r-o-w** [comment]
The StableDiffusion3 Pipelines inherit from [SD3LoraLoaderMixin](https://github.com/huggingface/diffusers/blob/38a3e4df926c59bc122191c0fc8066755e98b6d2/src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py#L131), which provide methods for loading loras. It should be as simple as:

```python
pipe.load_lora_weights("my-awesome-account-name/my-awesome-sd3-lora-name", adapter_name="sd3_lora")
pipe.set_adapters(["sd3_lora"], [0.8])  # set lora strength
```

You can find more information in our [docs](https://huggingface.co/docs/diffusers/en/api/loaders/lora#diffusers.loaders.SD3LoraLoaderMixin).
